metron-deployment/roles/metron_streaming/tasks/main.yml

```diff
@@ -51,4 +51,8 @@
 
 - include: hdfs_purge.yml
 
+- include: create_index_templates.yml
+  when: install_elasticsearch == True
```

Comment:
Due to METRON-180 we no longer use `install_elasticsearch`.  We will need to find another way to do this.  There are 8 different ways to do it.  Trying to think of the best way... hmmm.